advocacy_docs/partner_docs/ToadEdgeGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Toad Edge'
+---
+
+Implementing Quest Toad Edge with EDB Postgres Advanced Server or EDB Postgres Extended requires the following components: 
+
+- A running EDB Postgres Advanced Server or EDB Postgres Extended instance.
+- Quest Toad Edge application installed on a system where you will manage the running database. (EDB JDBC Driver included)
+
+The following steps show how to configure Quest Toad Edge for EDB Postgres Advanced Server and EDB Postgres Extended:
+
+**EDB Postgres Advanced Server Configuration:**
+
+1. Launch the Toad Edge application
+2. On the main menu, go the `Connect` dropdown and select `New Connection`
+3. In the `New Connection` screen, enter the following values: `Hostname`,`Port`,`Database`,`Username`,`Password`
+4. Select `EDB Postgres Advanced Server` on the left pane. 
+5. Click `Test Connection` to verify connectivity to the database server.
+6. If the connection is successful, click `Connect`. 
+
+<p align="center">
+  <img src="Images/EPASConfiguration.png">
+</p>
+
+You can update the EDB JDBC driver by downloading the latest version from EDB and then replacing the default EDB JDBC jar. 
+1. Download the new EDB JDBC Driver.
+2. Go to the drivers directory for your Toad Edge installation. For example: `C:\Program Files\Quest Software\Toad Edge\lib\drivers`
+3. Rename or backup the existing EDB JDBC Driver so that you have a copy to restore if needed.
+4. Copy the new EDB JDBC Driver to the `drivers` folder.
+5. Launch Toad Edge and create a new connection for EDB Postgres Advanced Server as shown above in the `EDB Postgres Advanced Server Configuration` section.
+
+
+
+**EDB Postgres Extended Configuration:**
+
+1. Launch the Toad Edge application
+2. On the main menu, go the `Connect` dropdown and select `New Connection`
+3. In the `New Connection` screen, enter the following values: `Hostname`,`Port`,`Database`,`Username`,`Password`
+4. Select `PostgreSQL` on the left pane. 
+5. Click `Test Connection` to verify connectivity to the database server.
+6. If the connection is successful, click `Connect`. 
+
```

Comment:
punctuation missing for step 1, 2, and 3.
step 2. go to the
step 6. is successful, click Connect. Otherwise verify your database information and try again.

These are the same edits from Advanced Server section as we probably used the same text.